File: Pemrograman 1/MODUL 4/Bahasa C/PRAK405-2410817110002-Akhmad_Daffa_Azzikri.c
#include <stdio.h>

int main() {
    int n, k, sum = 0;

    printf("");
    scanf("%d", &n);
    printf("");
    scanf("%d", &k);

    for (int i = 1; i <= n; i++) {
        int sum2 = 0;
        for (int j = 1; j <= i; j++) {
            sum2 += j * k;
            printf("(%d * %d) ", j, k);
            if (j < i) printf("+ ");
        }
        printf("= %d\n", sum2);
        sum += sum2;
    }
    printf("%d\n", sum);

    return 0;
}

File: Pemrograman 1/MODUL 4/Bahasa Python/PRAK405-2410817110002-Akhmad_Daffa_Azzikri.py
n, k = input("").split()
k = int (k)
n = int (n)

total_sum = 0
for i in range(1, n + 1):
    sum2 = sum(j * k for j in range(1, i + 1))
    for j in range(1, i + 1):
        print(f"({j} * {k})", end=" ")
        if j < i:
            print("+", end=" ")
    print(f"= {sum2}")
    total_sum += sum2

print(total_sum)